test/unit/Util/Autoload/ClassLoaderTest.php

```diff
@@ -61,6 +61,9 @@ public function testAutoloadTriggersLoaderMethod(): void
         spl_autoload_unregister($loader);
     }
 
+    /**
+     * @runInSeparateProcess
```

Comment:
Probably to be kept in the same process?

Reply:
Note: `@runInSeparateProcess` is to be avoided, if possible: we almost never use global state, so we should probably avoid it whenever possible.

Reply:
I think that `ClassLoader` uses global state: https://github.com/Roave/BetterReflection/blob/5.1.x/src/Util/Autoload/ClassLoader.php#L26

Reply:
Yes, hence why we unregister it? :thinking: 

Is it avoidable to run this in a separate process at all?

Reply:
Removed for now, it looks it works :)